### ISSUE TITLE
this.getDefaultTabAutocompleteModel is not a function

### ISSUE THREAD

**Issue by opyate** [continuedev/continue#995]
### Before submitting your bug report

- [ ] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [x] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [x] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

- OS: Linux Ubuntu 20.04.6
- Continue: 

latest from Changelog:

```
0.8.15 - 2024-03-05
```

More Info
```
Published 2023-05-27, 21:17:59
Last released 2024-03-18, 20:58:08
Last updated 2024-03-18, 21:53:54
Identifier continue.continue
```

- IDE:

```
Version: 1.87.1
Commit: 1e790d77f81672c49be070e04474901747115651
Date: 2024-03-06T00:21:07.287Z
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Linux x64 5.15.0-97-generic
```

### Description

Started seeing this toast in the lower right corner after every keypress:

```
this.getDefaultTabAutocompleteModel is not a function
```

![image](https://github.com/continuedev/continue/assets/96808/c8f41a78-8063-4f75-9a70-7bf64e31dc57)

### To reproduce

In any code file, start typing, and the toast pops up on every key press.

**Comment by byangtri** [comment]
Encountering same issue on MacOS

**Comment by sestinj** [comment]
@opyate @byangtri what version of Continue is this happening with?

**Comment by opyate** [comment]
@sestinj 

0.8.15 (just updated original post)

Latest from Changelog:

```
0.8.15 - 2024-03-05
Added
Tab-autocomplete in beta
```

Just saw that tap-autocomplete is in beta, so breakages/bugs are expected.

**Comment by byangtri** [comment]
> Contributor

v0.8.20

**Comment by sestinj** [comment]
@opyate @byangtri this is solved now in 0.8.21
